go/README.md

```diff
@@ -0,0 +1,99 @@
+# Omise golang example
+
+This is a example project about using Opn Payment.
+
+It has some examples.
+- Credit Card Payment
+- PayPay
+
+## Requirements
+To follow along, you will need:
+
+An Omise account (and your $OMISE_PUBLIC_KEY and $OMISE_SECRET_KEY)
+
+- golang 1.16
```

Comment:
This version is EOL as of March 2022.

Reply:
@danfowler 
Updated to 1.21
https://github.com/omise/examples/pull/32/commits/d846de0b3d8fed816b983ac34a8419b95ba6a282